web-app/src/services/clientService.js

```diff
@@ -7,27 +7,3 @@ export const createClient = client => {
 export const getAllClients = () => {
   return axios.get('/clients/');
 };
-
-export const getClientById = clientId => {
-  return axios.get(`/clients/${clientId}`);
```

Comment:
getClientById is being used for updating a client?

Reply:
Well spotted, sorry!